packages/flame/lib/src/components/isometric_tile_map_component.dart

```diff
@@ -105,32 +128,40 @@ class IsometricTileMapComponent extends PositionComponent {
 
   /// Same as getBlockRenderPosition but the arguments are exploded as integers.
   Vector2 getBlockRenderPositionInts(int i, int j) {
-    final halfTile = effectiveTileSize / 2;
+    final halfTile = Vector2(effectiveTileSize.x / 2, (effectiveTileSize.y / 2) / scalingFactor);
     final pos = Vector2(i.toDouble(), j.toDouble())..multiply(halfTile);
     return cartToIso(pos) - halfTile;
```

Comment:
This should preferably only create one new `Vector2` too, but since you didn't create more objects here than what was already created you don't have to do that if you don't want to.